src/commands/argument.js

```diff
@@ -97,6 +98,12 @@ class Argument {
 		 * @type {number}
 		 */
 		this.wait = typeof info.wait !== 'undefined' ? info.wait : 30;
+
+		/**
+		 * Whether the argument mentions the user when prompting
+		 * @type {boolean}
+		 */
+		this.reply = info.reply === undefined ? true : info.reply;
```

Comment:
You could just do !!this.reply

Reply:
wouldn't that make it false by default?

Reply:
It would just make it a boolean

Reply:
Actually, you're right, as it would be undefined by default.

Reply:
if the user doesn't specify reply, it would be undefined and coercing it would result in false, and i'm trying to make it true by default

Reply:
Still, `typeof info.reply === 'undefined' ? true : info.reply`

Reply:
👌